app/templates/components/currency-amount.hbs

```diff
@@ -1 +1 @@
-{{#if (gt @amount 0)}}{{currency-symbol @currency}}{{/if}} {{format-money @amount}}
+{{#if (gt @amount 0)}}{{currency-symbol @currency}}{{/if}} {{format-money this.modifiedAmount}}
```

Comment:
No need of using format-money once you use what I wrote in the issue comments. In fact, it will break the format-money helper. 11.000,23 is not a valid number